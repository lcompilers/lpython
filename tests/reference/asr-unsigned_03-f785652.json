{
    "basename": "asr-unsigned_03-f785652",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/unsigned_03.py",
    "infile_hash": "72dd2c6e17b137b6b9f66deeaab0af4ebe3044cb16009e0fab25f05a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-unsigned_03-f785652.stderr",
    "stderr_hash": "d90013a25d9aaa91fbbf2b1193cd06be94a4e716f0fe99771cde5601",
    "returncode": 2
}