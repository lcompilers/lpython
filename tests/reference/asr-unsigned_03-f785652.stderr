semantic error: The result of the unary minus `-` operation is negative, thus out of range for u64
 --> tests/errors/unsigned_03.py:3:7
  |
3 | print(-u64(u32(10)))
  |       ^^^^^^^^^^^^^ use -i64(u) for signed result
